### ISSUE TITLE
Too many levels of symbolic links

### ISSUE THREAD

**Issue by gukj** [samschott/maestral#469]
**Describe the bug**
When starting sync on a folder that contains a Final Cut Pro X project, an error referring to symbolic links stops the sync.

**To Reproduce**
Starting sync with a FCPX project of sufficient(?) size.

**Expected behaviour**
Should sync.

**System:**
 - Maestral version: 1.5.0
 - Python version: 2.7.16
 - OS: macOS 11.6

**Traceback**
```
Traceback (most recent call last):
File "Contents/Resources/app_packages/maestral/manager.py", line 791, in _handle_sync_thread_errors
File "Contents/Resources/app_packages/maestral/manager.py", line 769, in startup_worker
File "Contents/Resources/app_packages/maestral/sync.py", line 2836, in download_sync_cycle
File "Contents/Resources/app_packages/maestral/sync.py", line 2993, in apply_remote_changes
File "Contents/Resources/Support/Python/Resources/lib/python3.9/concurrent/futures/_base.py", line 608, in result_iterator
File "Contents/Resources/Support/Python/Resources/lib/python3.9/concurrent/futures/_base.py", line 445, in result
File "Contents/Resources/Support/Python/Resources/lib/python3.9/concurrent/futures/_base.py", line 390, in __get_result
File "Contents/Resources/Support/Python/Resources/lib/python3.9/concurrent/futures/thread.py", line 52, in run
File "Contents/Resources/app_packages/maestral/sync.py", line 3330, in _create_local_entry
File "Contents/Resources/app_packages/maestral/sync.py", line 3468, in _on_remote_folder
File "Contents/Resources/app_packages/maestral/sync.py", line 936, in update_index_from_sync_event
File "Contents/Resources/app_packages/maestral/sync.py", line 3231, in _get_ctime
File "Contents/Resources/app_packages/maestral/sync.py", line 3231, in _get_ctime
File "Contents/Resources/app_packages/maestral/sync.py", line 3231, in _get_ctime
[Previous line repeated 34 more times]
File "Contents/Resources/app_packages/maestral/sync.py", line 3230, in _get_ctime
OSError: [Errno 62] Too many levels of symbolic links: '/Users/gustavkjellin/Dropbox/GKDC/_OPEN_PROJECTS/TEMPLATE/03_TEMPLATE_happy/_TEMPLATE_HAPPY_marketing/WISHBONE_TEMPLATE_movie.fcpbundle/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache/.fcpcache'
```

**Comment by SamSchott** [comment]
Yes, symbolic links are not properly supported at the moment. Part of the problem is that the public Dropbox API does not allow creating symbolic links on Dropbox servers, unlike the private API. I've submitted a feature request but they don't share their future plans or timelines with developers.

**Comment by SamSchott** [comment]
For your use case, it's probably best to go back to the official client...

**Comment by FransUrbo** [comment]
Could we instead of crashing the daemon, instead just skip this link and log this as an (sync) error?

That should be easy enough, right?

**Comment by SamSchott** [comment]
> Could we instead of crashing the daemon, instead just skip this link and log this as an (sync) error?

Yes, that is certainly the better behaviour. This should have been fixed by #508 and will be available with the next release / update.

However, the current treatment of symlinks is not ideal and Maestral will under certain circumstances replace them with an actual folder and its content. Unfortunately, until it is possible to explicitly create symlinks through the public API, this appears to be the better alternative.

**Comment by samschott** [comment]
Closing since symlinks are now handled properly.
